dashboard/test/ui/runner.rb

```diff
@@ -240,8 +242,7 @@ def run_tests(arguments)
 features_to_run = passed_features.empty? ? all_features : passed_features
 browser_features = $browsers.product features_to_run
 
-git_branch = `git rev-parse --abbrev-ref HEAD`.strip
-ENV['BATCH_NAME'] = "#{git_branch} | #{Time.now}"
+ENV['BATCH_NAME'] = "#{GIT_BRANCH} | #{Time.now}"
```

Comment:
There is also now a `GitUtils.current_branch` https://github.com/code-dot-org/code-dot-org/blob/staging/lib/cdo/git_utils.rb#L40-L40